ocp_resources/image_content_source_policy.py

```diff
@@ -7,3 +7,21 @@ class ImageContentSourcePolicy(Resource):
     """
 
     api_group = Resource.ApiGroup.OPERATOR_OPENSHIFT_IO
+
+    def __init__(self, repository_digest_mirrors=None, **kwargs):
+        """
+        Create/Manage ICSP configuration object. API reference:
+        https://docs.openshift.com/container-platform/4.12/rest_api/operator_apis/imagecontentsourcepolicy-operator-openshift-io-v1alpha1.html
+
+        Args:
+            repository_digest_mirrors (list): [{<source>: <mirrors>}, ..., {<source>: <mirrors>}]
```

Comment:
- `repository_digest_mirrors` is mandatory, please raise if not passed
- please explain what is the format of `repository_digest_mirrors` - should be a list of dictionaries, each containing `mirrors` key with a list of mirrors and a `source` key which contains a source str

```
[{'mirrors': ['mirror1',],  'source': 'src'}]

```

Reply:
done